src/pages/home/report/ReportActionPropTypes.js

```diff
@@ -10,7 +10,10 @@ export default {
     person: PropTypes.arrayOf(ReportActionFragmentPropTypes).isRequired,
 
     // ID of the report action
-    sequenceNumber: PropTypes.number.isRequired,
+    sequenceNumber: PropTypes.oneOfType([
+        PropTypes.number,
+        PropTypes.string,
```

Comment:
Should we maybe always cast these to a string?

Reply:
Just curious, at what point should we cast them to a string? And why? I didn't do anything about this because I'm not too sure why it would be preferable to cast to a string, but maybe I'm missing something obvious.